### ISSUE TITLE
Support local and remote URLs for the same instance

### ISSUE THREAD

**Issue by sagebind** [cassidyjames/butler#34]
The mobile companion apps allow you to specify a local / internal URL as well as a normal / remote URL. When your device is on WiFi using the SSID you specify, it will load the local URL. Otherwise, it will load the remote URL.

It would be great if Butler could support this kind of option as well. Currently I am always using my Nabu Casa remote URL on my laptop so that it works anywhere, even though it _could_ connect directly while connected to WiFi to avoid the extra latency and bandwidth restrictions.

**Comment by cassidyjames** [comment]
Yep, excellent idea! This is a good example of a unique value-add that a native app can add compared to a web browser.
